pydoover/cloud/processor/application.py

```diff
@@ -94,6 +96,9 @@ async def _setup(self, initial_payload: dict[str, Any]):
             # if there's no config defined this can legitimately be None in which case don't bother.
             self.config._inject_deployment_config(data["deployment_config"])
 
+        # Store the deployment config for later use
+        self.received_deployment_config = data["deployment_config"]
```

Comment:
Is this ever designed to be used by the user? Looks like it's only used to get the app name - would it be better to just get the app name here? Generally people should be using `self.config.xxxx.value` to access config rather than the raw dict (I know that's not possible here)

ie. 
```
self.app_name = data["deployment_config"].get("APP_DISPLAY_NAME")
```
... not sure if there's a case where `deployment_config` would be `None` (need to check what doover data returns when it can't find a channel, can't remember if it's None (Value::Null) or object

Reply:
Its needed if any of the capitalised values are to be accessed, so in the latest commit I used it to get DEVICE_LIST